nsfc-plugin/pom.xml

```diff
@@ -0,0 +1,798 @@
+<!--
+    Copyright (c) Intel Corporation
+    Copyright (c) 2017
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+	<parent>
+	    <groupId>org.osc.plugin</groupId>
+		<artifactId>sdn-controller-nsfc-plugin</artifactId>
+		<version>2.0.1-SNAPSHOT</version>
+	</parent>
+
+    <groupId>org.osc.plugin</groupId>
+    <artifactId>nsfc-plugin</artifactId>
+    <name>org.osc.plugin:nsfc-plugin</name>
+
+    <properties>
+        <powermock.version>1.6.5</powermock.version>
+        <junit.version>4.12</junit.version>
+        <mockito.version>1.10.19</mockito.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>osc-nexus</id>
+            <name>OSC Nexus</name>
+            <url>http://ci.opensecuritycontroller.org:8082/nexus/content/repositories/releases</url>
+        </repository>
+        <repository>
+            <id>osc-nexus-snapshot</id>
+            <name>OSC Nexus</name>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <url>http://ci.opensecuritycontroller.org:8082/nexus/content/repositories/osc-snapshot</url>
+        </repository>
+    </repositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>1.7.25</version>
+            <scope>provided</scope>
+        </dependency>
+         <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.2.3</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.osc.plugin</groupId>
+            <artifactId>nsfc-uber-openstack4j</artifactId>
+            <version>2.0.1-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
why are these dependencies needed? Shouldnt all of this be encapsulated within uber-openstack4j?